docs/source/contributing.rst

```diff
@@ -51,6 +51,9 @@ Contribution workflow
 
 The ``rail`` and ``rail_<xxx>`` repositories use an issue-branch-review workflow, 
 similar to the standard `GitHub Flow <https://docs.github.com/en/get-started/quickstart/github-flow>`_.
+We typically use ``git`` as our version control tool, there are many resources
```

Comment:
```suggestion
We typically use ``git`` as our version control tool. There are many resources
```
Small grammar nit (this could stay a comma as long as we drop a conjunction right after)